test/hooks/achievement-rule-checker.test.js

```diff
@@ -194,4 +194,59 @@ describe('\'achievement-rule-checker\' hook', () => {
 
     assert.deepEqual(result[0].amount, 2);
   });
+
+  it('gives maxAwarded achievements at once', async () => {
```

Comment:
same here

Reply:
In my opinion more tests don't do any harm, so I would probably just leave it in as it is wanted behavior, although not the standard case.